dev-python/yapf/yapf-0.29.0.ebuild

```diff
@@ -0,0 +1,24 @@
+# Copyright 2018-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{5,6,7} )
```

Comment:
Please drop drop py2.7 and py3.5 support. I know upstream supports py2.7, but it does state some features will not work properly. Additionally, Gentoo is beginning to remove py2 support.  py3.5 was skipped as well.